tests/foreman/cli/test_syncplan.py

```diff
@@ -491,3 +491,18 @@ def test_positive_delete_1(self, test_data):
             0,
             "Expected an error here"
         )
+
+    @skip_if_bug_open('bugzilla', 1261122)
+    def test_bz1261122_enabled_state_visible(self):
+        """@Test: Check if Enabled field is displayed in sync-plan info output
+
+        @Feature: Sync Plan Info
+
+        @Assert: Sync plan Enabled state is displayed
+
+        """
+
+        new_sync_plan = self._make_sync_plan()
+        # Fetch the info
```

Comment:
Nitpick: drop this comment. It doesn't explain much, and the code itself is already clear and concise.